File: 6825/main.py
m = float(input())
k = float(input())
bmi = m/(k*k)
print(m,k,bmi)

if bmi >= 25:
    print("Overweight")
elif bmi >= 18.5 and bmi <= 25.0:
    print("Normal weight")
elif bmi <= 18.5:
    print("Underweight")

File: 6825/main.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
    double w,h;
    cin >> w >> h;
    double bmi = w/pow(h,2);
    if (bmi > 25) cout << "Overweight" << "\n";
    else if (bmi >= 18.5) cout << "Normal weight" << "\n";
    else cout << "Underweight" << "\n";
    
}